build.gradle

```diff
@@ -31,7 +31,7 @@ task clean(type: Delete) {
 
 ext {
     // Sdk and tools
-    minSdkVersion = 15
+    minSdkVersion = 16
```

Comment:
@therajanmaurya I have changed `minSdkVersion ` to 16, so that I can use `android:fontFamily` in styles.xml

Reply:
ok